docs/src/_guide/advanced-topics.md

```diff
@@ -13,3 +13,159 @@ Most of Meltano's features are available without installing any additional packa
 * `s3` - Support for using S3 as a [state backend](/concepts/state_backends)
 * `gcs` - Support for using Google Cloud Storage as a [state backend](/concepts/state_backends)
 * `azure` - Support for using Azure Blob Storage as a [state backend](/concepts/state_backends)
+
+## Airbyte Connector Integration FAQ
+
+### How do the Singer and Airbyte specifications relate?
+
+The Singer specification was started in 2016 by Stitch Data. It specified a data transfer format that would allow any number of data systems, called taps, to send data to any data destination, called targets. Airbyte was incorporated in 2020 and created their own specification that was heavily inspired by Singer. There are differences but the core of each specification is sending new-line delimited JSON data from STDOUT of a tap to STDIN of a target.
+
+### How does the Airbyte Connector with Meltano integration work?
+
+A community member used the Meltano Singer SDK to write tap-airbyte-wrapper. This wrapper connector calls the Docker image for a given Airbyte Source and translates the messages into a Singer-compatible format. The output of the tap is conformed to the Singer standard and then can be sent to any Singer target, many of which are listed on [MeltanoHub](hub.meltano.com/loaders/).
+
+### How do I get help if I need it?
+
+We first recommend you read through this FAQ to see if your question has been answered. We next recommend searching the [Meltano docs](https://docs.meltano.com) to see if your answer is there.
+
+If you still need help we’d recommend either filing an issue in the [Meltano Github repository](https://github.com/meltano/meltano/issues) or join our [Slack community](https://meltano.com/slack) to ask there.
+
+We also have [weekly Office Hours](https://www.addevent.com/calendar/Li390615) where you can talk to the Meltano team and ask any questions you have to us!
+
+### Does this wrap the Airbyte UI or API?
```

Comment:
```suggestion
### Do I need to have an Airbyte UI or API instance to use this?
```

I think the main point this should address is whether someone needs to run Airbyte and Meltano to use this. We want them to know that Meltano is directly running the connectors without the Airbyte interface.